AWSKinesis/AWSAbstractKinesisRecorder.m

```diff
@@ -263,7 +263,7 @@ - (AWSTask *)submitAllRecords {
                                                   @"stream_name": [rs stringForColumn:@"stream_name"],
                                                   }];
 
-                    [partitionKeys addObject:[rs stringForColumn:@"partition_key"]];
+                    [partitionKeys addObject:[rs stringForColumn:@"rowid"]];
```

Comment:
The changes look great. Before we can accept this merge, could you re-name the object appropriately so that the flow makes more sense? `partionKeys` should change to `rowIds` ideally.